_episodes/l3-01-guis-ides.md

```diff
@@ -61,7 +61,7 @@ section.
 So far, your `recipe` repository is fairly simple, but it is worth noting that
 repositories can become complex, e.g.:
 
-![]({{ site.baseurl }}/fig/gui_complex_repo.png "Example of a complex repository")
+![Example of a complex repository]({{ site.baseurl }}/fig/gui_complex_repo.png "Example of a complex repository")
```

Comment:
I *thought* I was correctly specifying an alttext here... Is there a difference between having the alttext in the square brackets or in quotes after the path? I'm guessing we don't need to do it both ways.

Reply:
(Same for image tags below)

Reply:
PS -- Not suggesting you change all these if specifying it twice doesn't matter

Reply:
So it seems (https://www.w3schools.io/file/markdown-images/) that the text in square brackets is the alt text, which gets picked up by screen readers and if the image fails to load, while the text in quotes after the path is the image title, which is shown if you hover your cursor over the image.

Reply:
Oh, how odd. I thought those two things were the same. TIL!